wpilibc/src/main/native/cpp/drive/DifferentialDrive.cpp

```diff
@@ -170,10 +179,6 @@ void DifferentialDrive::InitSendable(wpi::SendableBuilder& builder) {
   builder.SetSmartDashboardType("DifferentialDrive");
   builder.SetActuator(true);
   builder.SetSafeState([=, this] { StopMotor(); });
-  builder.AddDoubleProperty(
-      "Left Motor Speed", [=, this] { return m_leftMotor->Get(); },
-      [=, this](double value) { m_leftMotor->Set(value); });
-  builder.AddDoubleProperty(
-      "Right Motor Speed", [=, this] { return m_rightMotor->Get(); },
-      [=, this](double value) { m_rightMotor->Set(value); });
+  builder.AddDoubleProperty("Left Motor Speed", nullptr, m_leftMotor);
+  builder.AddDoubleProperty("Right Motor Speed", nullptr, m_rightMotor);
```

Comment:
Just to confirm, this loss of reporting is fine?

Reply:
Would recording the previously set values and returning those to Sendable be acceptable?

Reply:
Yes that would work.  The getter is called every loop (with the return value being sent to NT) if the Sendable is provided to SmartDashboard.putData or similar.